config/application.rb

```diff
@@ -101,5 +101,7 @@ class Application < Rails::Application
     config.conversation_js_progressive_disclosure_delay = nil
 
     config.bigquery_dataset_id = ENV["BIGQUERY_DATASET"]
+
+    config.answer_strategy = ENV["ANSWER_STRATEGY"]
```

Comment:
```suggestion
    config.answer_strategy = ENV.fetch("ANSWER_STRATEGY", "openai_structured_answer")
```

so there is a default set automatically.